compiler/plugins/target/AMD-AIE/iree-amd-aie/IR/AMDAIEDialect.cpp

```diff
@@ -8,6 +8,10 @@
 
 #include "iree-amd-aie/IR/AMDAIEAttrs.h"
 #include "iree-amd-aie/IR/AMDAIEDialect.cpp.inc"
+#include "iree-amd-aie/IR/AMDAIETypes.cpp.inc"
```

Comment:
Typically these are kept separately from the actual .h includes cause these are tablegen generated . See examples here https://github.com/openxla/iree/blob/954cb36d8201638823a08d0c65eb35325758c730/compiler/src/iree/compiler/Dialect/HAL/IR/HALTypes.h#L30

Reply:
I think I got what you mean and I addressed this here: https://github.com/nod-ai/iree-amd-aie/pull/270